projects/plugins/jetpack/_inc/client/recommendations/product-card-upsell/index.jsx

```diff
@@ -65,48 +90,70 @@ const ProductCardUpsell = ( {
 					<li key={ feature }>{ feature }</li>
 				) ) }
 			</ul>
-			<div className="jp-recommendations-product-card-upsell__price">
-				<div className="jp-recommendations-product-card-upsell__raw-price">
-					<h2>
-						<sup className="jp-recommendations-product-card-upsell__currency-symbol">
-							{ currencyObject.symbol }
-						</sup>
-						<span className="jp-recommendations-product-card-upsell__price-integer">
-							{ currencyObject.integer }
-						</span>
-						<sup className="jp-recommendations-product-card-upsell__price-fraction">
-							{ currencyObject.fraction }
-						</sup>
-					</h2>
-				</div>
-				<div className="jp-recommendations-product-card-upsell__billing-time-frame">
-					{ `${ cost_timeframe }, ${ billing_timeframe }` }
-				</div>
-			</div>
+			{ ! ( isFetchingDiscount || isFetchingOffers ) && (
```

Comment:
If this data is still loading, is there a loading state that can be shown?

Reply:
I've added handling this loading state in a follow-up task.